src/hash_test.c

```diff
@@ -44,6 +44,21 @@ void testcase_blake2b_224(const char* inputHex, const char* expectedHex)
 	EXPECT_EQ_BYTES(expectedBuffer, outputBuffer, expectedSize);
 }
 
+void testcase_blake2b_160(const char* inputHex, const char* expectedHex)
```

Comment:
I'd also add a test to verify that the asset fingerprints produced are up to specs